lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -4268,69 +4257,58 @@ hasReturnCollateral (Cardano.Tx (Cardano.TxBody content) _) =
         Cardano.TxReturnCollateralNone -> False
         Cardano.TxReturnCollateral _ _ -> True
 
--- | Consistent pair of ProtocolParameters of both wallet and cardano-api types.
---
--- We try to use similar parameters to mainnet where it matters (in particular
+-- | We try to use similar parameters to mainnet where it matters (in particular
 -- fees, execution unit prices, and the cost model.)
---
--- NOTE: We don't have a 'Cardano.ProtocolParameters -> ProtocolParameters'
--- function, so we need to manually ensure the hard-coded values are consistent.
-mockProtocolParametersForBalancing
-    :: (ProtocolParameters, Cardano.ProtocolParameters)
```

Comment:
No more "two sources of truth" 🎉

Reply:
(There are some other tests in this module still using `mockProtocolParameters` which I'd like to question completely and/or migrate, but that's a separate problem)